unittests/test_data_spec_generator.py

```diff
@@ -1170,7 +1170,7 @@ def test_write_structure(self):
         with self.assertRaises(ParameterOutOfBoundsException):
             self.dsg.write_structure(-1)
         with self.assertRaises(ParameterOutOfBoundsException):
-            self.dsg.write_structure(16)
+            self.dsg.write_structure(32)
```

Comment:
And then this change is probably not needed either

Reply:
Oh, now I see what this is doing too - it tests out of bounds exception, so not a big deal!